.github/workflows/ci.yml

```diff
@@ -30,6 +30,28 @@ jobs:
           command: fmt
           args: --check
 
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+
+      - name: Cache
+        uses: Swatinem/rust-cache@v2
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Check each crate
```

Comment:
@ascjones do you mind making a small follow-up explaining why we need this check?

For someone just reading the CI file they wouldn't have enough context to know why we manually check all crates